packages/devtools/lib/src/timeline/timeline.dart

```diff
@@ -23,16 +23,16 @@ import 'timeline_protocol.dart';
 // Light mode is Light Blue 50 palette and Dark mode is Blue 50 palette.
 // https://material.io/design/color/the-color-system.html#tools-for-picking-colors.
 const mainUiColorLight = Color(0xFF81D4FA); // Light Blue 50 - 200
-const mainUiColorSelectedLight = Color(0xFF0091EA); // Light Blue 50 - A700
+const mainUiColorSelectedLight = Color(0xFFD4D7DA);
```

Comment:
nit: Maybe mention Ui selection is a lighter gray and Gpu selection is a darker gray?

Reply:
done.